scio-bigquery/src/main/scala/com/spotify/scio/bigquery/types/ConverterProvider.scala

```diff
@@ -250,7 +255,9 @@ private[types] object ConverterProvider {
     // =======================================================================
 
     def cast(tree: Tree, tpe: Type): Tree = {
+      val provider: ValidationProvider = ValidationProviderFinder.getProvider
       tpe match {
+        case t if provider.shouldOverrideType(c)(t) => q"$tree.toString"
```

Comment:
Is this always string? Or should we be getting it from the class instead?

Reply:
yeah in this situation we need a primitive type if you look at the `DateTime` examples we're printing it as a string format

we don't strictly need a string but I think it's ok to require it in this case